src/db_queries/expenses_paid.ts

```diff
@@ -0,0 +1,52 @@
+import { Helpers } from '../utils/helpers';
+import { Request, Response } from 'express';
+import dbClient from "../db/client";
+import CenterExpensePaid from "../models/center_expense_paid.model";
+
+const helpers = new Helpers();
+
+export class ExpensesPaid {
+
+    private centerExpensesPaidQuery = "SELECT * FROM organizations_centers_expenses_paid WHERE center_id = $1 AND month = $2 AND year = $3";
+    private updateExpensePaidQuery = "UPDATE organizations_centers_expenses_paid SET amount = $1 WHERE center_id = $2 AND month = $3 AND year = $4";
+    private createExpensePaidQuery = "INSERT INTO organizations_centers_expenses_paid (center_id, month, year, amount) VALUES ($1, $2, $3, $4)";
+
+    constructor() {
+        helpers.autoBind(this);
+    }
+
+    async getCenterExpensePaid(request: Request, response: Response): Promise<void>{
+        const {center_id} = request.params;
+        const {month, year} = request.query
+        try {
+            const result = await dbClient.query<CenterExpensePaid>(this.centerExpensesPaidQuery, [center_id, month, year]);
+
+            if(result.rowCount === 0) {
+                response.status(404).send("No expense paid yet");
```

Comment:
We should not throw an error here. If no expense has been paid yet, then it is not an error, and we can pass an empty array.